gitpod-io/website
# [Homepage] Make card links clickable

## Description



Make the Homepage titles card clickable w/ No style change on hover/focus. It would just change the cursor to a pointer when the user hovers over those cards.

<img width="1332" alt="image" src="https://github.com/gitpod-io/website/assets/55068936/dae32895-e08f-4ced-b7c5-9e92dd31aacb">

## How to test



- Open [preview](https://deploy-preview-3932--gitpod-kumquat.netlify.app/)
- :eyes: See titles cards and hover on it :)) 

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing


## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.


</details>
